include/class_loader/visibility_control.hpp

```diff
@@ -61,4 +61,11 @@
   #define CLASS_LOADER_PUBLIC_TYPE
 #endif
 
+// based on wiki.ros.org: http://wiki.ros.org/win_ros/Contributing/Dll%20Exports
+// Ignore warnings about import/exports when deriving from std classes.
```

Comment:
@kejxu are there documentation permalinks you can add linking to the documentation for the disabled error classes?

Reply:
not sure what this would mean, what kind of links are documentation permalinks? we are just copying the comment from the linked ros wiki page, section 3, "What if I Don't Want Cpp_Common as a Dependency?"

Reply:
I recognize that it's coming from the wiki. Is the link https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4251 and https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-2-c4275 stable enough that if we wanted to add them as breadcrumbs describing the error being supressed.

Reply:
thanks for sharing the links! there is no guarantee that the links would be valid forever; however, msdn/docs.microsoft.com links generally remain valid for very long time, making it safe to include them in ros docs (in case these links are to be added into [ros wiki](http://wiki.ros.org/win_ros/Contributing/Dll%20Exports)).

if, for some reason, these compiler documentations go invalid (which I believe would never happen...), there are multiple channels we could report this issue and get a new link

Reply:
```suggestion
// Ignore warnings about import/exports when deriving from std classes.
// https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4251
// https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-2-c4275
```

The wiki includes an excerpt from cpp_common. I'm not a maintainer there, but I'd prefer these docs linked in our visibility control header.